kthfsdv_exc2.py

```diff
@@ -0,0 +1,72 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import sys
+import os
```

Comment:
```suggestion
import os
import sys

import matplotlib.pyplot as plt
import numpy as np
```

PEP-8 suggestion :slightly_smiling_face: